<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="19360" started-at="2019-10-09T11:10:36Z" finished-at="2019-10-09T11:10:55Z">
    <groups>
    </groups>
    <test name="FirefoxTest" duration-ms="49" started-at="2019-10-09T11:10:36Z" finished-at="2019-10-09T11:10:36Z">
      <class name="com.OH.e2e.Script">
        <test-method status="FAIL" signature="LaunchApplication(java.lang.String)[pri:1, instance:com.OH.e2e.Script@5622fdf]" name="LaunchApplication" duration-ms="12" started-at="2019-10-09T16:40:36Z" finished-at="2019-10-09T16:40:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.OH.e2e.Script.LaunchApplication(Script.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchApplication -->
      </class> <!-- com.OH.e2e.Script -->
    </test> <!-- FirefoxTest -->
    <test name="IETest" duration-ms="19360" started-at="2019-10-09T11:10:36Z" finished-at="2019-10-09T11:10:55Z">
      <class name="com.OH.e2e.Script">
        <test-method status="PASS" signature="Create()[pri:0, instance:com.OH.e2e.Script@598067a5]" name="Create" is-config="true" duration-ms="134" started-at="2019-10-09T16:40:36Z" finished-at="2019-10-09T16:40:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Create -->
        <test-method status="FAIL" signature="LaunchApplication(java.lang.String)[pri:1, instance:com.OH.e2e.Script@598067a5]" name="LaunchApplication" duration-ms="19334" started-at="2019-10-09T16:40:36Z" finished-at="2019-10-09T16:40:55Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <exception class="org.apache.poi.EmptyFileException">
            <message>
              <![CDATA[The supplied file was empty (zero bytes long)]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.apache.poi.EmptyFileException: The supplied file was empty (zero bytes long)
	at org.apache.poi.util.IOUtils.peekFirstNBytes(IOUtils.java:65)
	at org.apache.poi.util.IOUtils.peekFirst8Bytes(IOUtils.java:48)
	at org.apache.poi.ss.usermodel.WorkbookFactory.create(WorkbookFactory.java:177)
	at org.apache.poi.ss.usermodel.WorkbookFactory.create(WorkbookFactory.java:144)
	at com.OH.libraries.Excel_Libraries.Excel_Reporter(Excel_Libraries.java:61)
	at com.OH.libraries.Utility_Libraries.Pass_Reporter(Utility_Libraries.java:82)
	at com.OH.e2e.Script.LaunchApplication(Script.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- org.apache.poi.EmptyFileException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchApplication -->
      </class> <!-- com.OH.e2e.Script -->
    </test> <!-- IETest -->
  </suite> <!-- Suite -->
</testng-results>
